lib/ReactViews/Map/Panels/SharePanel/SharePanel.tsx

```diff
@@ -168,6 +167,7 @@ class SharePanel extends React.Component<PropTypes, SharePanelState> {
         viewState={this.props.viewState}
         btnTitle={btnTitle}
         isOpen={this.state.isOpen}
+        //@ts-expect-error - not yet ready to tackle tsfying StorySharePanel
```

Comment:
Should this be 5 lines earlier (before the `<StorySharePanel`) so it is consistent with the placement for the ts-expect-error for MenuPanel?

Reply:
Those are slightly different errors, this one is specifically on the prop `onOpenChanged` and TS requires the expect comment to precede it immediately.

Reply:
I misunderstood what triggered it, what you describe is exactly what I wanted, thank you!